### ISSUE TITLE
GitHub template for New Issues

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1172]
The current GitHub template for New Issues is irrelevant to the Documentation repo. Let's change it. I want to lower barriers to contribute while collecting minimal useful actionable information.

GitHub has a couple of nice features, including a "chooser" and "forms".

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Examples with Sphinx:

- https://github.com/sphinx-doc/sphinx/issues/new/choose
- https://github.com/sphinx-doc/sphinx/issues/new?assignees=&labels=bug&template=bug-report.yml
- https://github.com/sphinx-doc/sphinx/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=%3Cshort+description+for+the+feature%3E

The forms can be overly complex so I don't know if that is the right way forward.

Here is our current default New Issue template.

```
* **I'm submitting a ...**
  - [ ] bug report
  - [ ] feature request


* **Do you want to request a *feature* or report a *bug*?**


* **What is the current behavior?**


* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**


* **What is the expected behavior?**


* **What is the motivation / use case for changing the behavior?**


* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)
```

**Comment by ksuess** [comment]
+1 for one single template ( no chooser)

My wishlist:
- radio buttons for
  - bug / error report
  - missing section / missing chapter report
- radio buttons for
  I am referencing:
  - docs.plone.org
  - 6-dev


I like to have links to community.plone.org and Discord, similar as https://github.com/sphinx-doc/sphinx/issues/new/choose

![image](https://user-images.githubusercontent.com/1144737/152470911-ec9bbc04-9f6c-45d5-9e0f-1f5ea886740f.png)

**Comment by ksuess** [comment]
BTW the "new PR" template is very good:
https://github.com/plone/documentation/blob/5.2/.github/PULL_REQUEST_TEMPLATE.md
It just needs an update to point to the new contributor guideline.

**Comment by ksuess** [comment]
ideally this would open a new issue WITH the new template:
![image](https://user-images.githubusercontent.com/1144737/152471246-d2272ecd-bcd3-42a6-989c-f92b0089aad5.png)

**Comment by stevepiercy** [comment]
> ideally this would open a new issue WITH the new template:

See https://github.com/plone/documentation/issues/1173

We can't easily do much with this, but it looks like the authors of the theme are very interested in improving the situation.

**Comment by stevepiercy** [comment]
> BTW the "new PR" template is very good: https://github.com/plone/documentation/blob/5.2/.github/PULL_REQUEST_TEMPLATE.md It just needs an update to point to the new contributor guideline.

See https://github.com/plone/documentation/issues/1174

**Comment by stevepiercy** [comment]
> +1 for one single template ( no chooser)
> 
> I like to have links to community.plone.org and Discord, similar as https://github.com/sphinx-doc/sphinx/issues/new/choose

The chooser allows the options for the Forum and Discord. Did you mean allow a chooser, but have only one choice to create an issue? If so, then I agree with that.

Here's what I think we need at the very least, incorporating your wish list slightly differently.

1. Plone Documentation versions affected: (checkboxes) (The issue could affect multiple versions. I omitted 4.x as I assume we will not support it once 6.x is released.)
    - 5.x
    - 6.x
2.  I am reporting: (radios)
    - A correction in content is needed (typographical error, misspelling, broken link, outdated or incorrect content)
    - Missing content
    - Automation, build, testing, or deployment issue
    - Suggestion to enhance or improve the documentation
    - Other
3. Description: (textarea)
    If relevant, please include links to the affected resources, screenshots, and any other helpful information.

-----

For the Chooser, here's what I suggest.

1. Report an issue with Plone Documentation (error, missing content, feature request)
2. Get support for Plone in the Community Forum (link to https://community.plone.org/)
3. Chat with the Documentation Team on Discord (link to https://discord.gg/zFY3EBbjaj)

Feedback welcome and appreciated.

**Comment by ksuess** [comment]
Perfect.

**Comment by spereverde** [comment]
I think it's OK, I agree with Katja to keep things as simple as possible,  if you're creating an issue, you don't want to read a lexicon first

I would not know what to choose if you want to add documentation from above?

**Comment by stevepiercy** [comment]
> I would not know what to choose if you want to add documentation from above?

What would make it clear? Here's the proposed choice:

> 1. Report an issue with Plone Documentation (error, missing content, feature request)

**Comment by spereverde** [comment]
The chooser is fine, since there is only 1 option to file an issue

I meant with the "I am reporting" radios
What is that information used for?

**Comment by nileshgulia1** [comment]
> +1 for one single template ( no chooser)
> 
> My wishlist:
> 
> * radio buttons for
>   
>   * bug / error report
>   * missing section / missing chapter report
> * radio buttons for
>   I am referencing:
>   
>   * docs.plone.org
>   * 6-dev
> 
> I like to have links to community.plone.org and Discord, similar as https://github.com/sphinx-doc/sphinx/issues/new/choose
> 
> ![image](https://user-images.githubusercontent.com/1144737/152470911-ec9bbc04-9f6c-45d5-9e0f-1f5ea886740f.png)

+1 from me. We can confine to a single template. Its less likely for developer to differentiate between bugs and feature when it comes to writing docs.

**Comment by stevepiercy** [comment]
> I meant with the "I am reporting" radios What is that information used for?

I see what you mean. That information can go into the Description. Let's remove the "I am reporting" radios.

From previous docs issues reported, the greatest confusion I see is (1) what version of Plone, and (2) people say "Section Title has incorrect information", and in the issue report the title is not correct and lacks a link to understand what the heck the user is talking about. I think that requiring a version checkbox selection and a Description is sufficient.

**Comment by stevepiercy** [comment]
Closed by #1175
